libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -76,8 +75,8 @@ class Compiler {
 
   // Adds an implicit macro definition obeyed by subsequent CompileShader()
   // calls.
-  void AddMacroDefinition(const string_piece& macro,
```

Comment:
This comment is still unaddressed.
Update blurb, and add tests for the nullptr case.